test/runtests.jl

```diff
@@ -17,6 +17,8 @@ module_tests = [
     :Unwrap => ["unwrap.jl"]
 ]
 
+!(dirname(@__FILE__) in LOAD_PATH) && push!(LOAD_PATH, dirname(@__FILE__))
```

Comment:
```suggestion
!(@__DIR__ in LOAD_PATH) && push!(LOAD_PATH, @__DIR__)
```
?
But why is this needed anyway?

Reply:
But shouldn't the individual test files (all) do this? Those that have `using FilterTestHelpers`, that is. You've amended some to be runnable individually, I think, but that would also require this `LOAD_PATH` manipulation, no? But then it doesn't have to go into `runtests.jl`. Or am I missing something?